mlflow/server/js/src/experiment-tracking/pages/experiment-evaluation-datasets/components/ExperimentEvaluationDatasetsIdCell.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Tag, Tooltip } from '@databricks/design-system';
+import type { Row } from '@tanstack/react-table';
+import type { EvaluationDataset } from '../types';
+
+export const DatasetIdCell = ({ row }: { row: Row<EvaluationDataset> }) => {
+  const datasetId = row.original.dataset_id;
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  const handleClick = () => {
+    navigator.clipboard.writeText(datasetId);
+    setShowTooltip(true);
+    setTimeout(() => {
+      setShowTooltip(false);
+    }, 3000);
+  };
+
+  return (
+    <Tooltip
+      content={showTooltip ? 'Copied!' : 'Click to copy'}
+      open={showTooltip ? true : undefined}
```

Comment:
nit: does it need to be undefined, or is it fine if it's `false`? 
```suggestion
      open={showTooltip}
```

Reply:
oh i see, we want to dynamically change it so that when the user hovers, it shows "Click to copy", and when they click it shows "Copied!" and stays open? if that's the case then ignore this and the above comment